cypress-tests/cypress/e2e/admin/pageBuilder/category/pageCategoriesCrud.cy.ts

```diff
@@ -0,0 +1,96 @@
+import { customAlphabet } from "nanoid";
+
+context("Categories Module", () => {
+    const nanoid = customAlphabet("abcdefghijklmnopqrstuvwxyz");
+    const category_name = nanoid(6);
+    const category_name_edited = nanoid(6);
+    const category_slug = nanoid(6);
+    const category_url = "/" + nanoid(6) + "/";
+    const category_url_edited = "/" + nanoid(6) + "/";
+
+    beforeEach(() => {
+        cy.login();
+        cy.pbDeleteAllCategories();
+    });
+
+    afterEach(() => {
+        cy.pbDeleteAllCategories();
+    });
+
+    it("Should be able to create, edit, and immediately delete a category", () => {
+        cy.visit("/page-builder/categories");
+
+        //Creates new category.
```

Comment:
```suggestion
        // Creates new category.
```

Please apply the same below.